elastic/integrations
# [Azure OpenAI] Map the source.* fields to ECS fields

This PR maps the source.* fields to ECS fields.

## Checklist



## How to test this PR locally

- Verify the source fields are mapped to the ECS fields.
- Run pipeline tests.
